internal/load-test/query_load.py

```diff
@@ -9,7 +9,7 @@
 from internal.core.keys import keys_show
 from internal.modules.bank.query import query_balances
 from internal.modules.staking.query import (
-    query_delegator_delegations,
+    query_delegator_delegation,
     query_staking_validators,
```

Comment:
```suggestion
```

Reply:
@anilCSE, can we replace that with `query_validator_set` method.

Reply:
Yes, let do that